### DISCUSSION TITLE
Feature to open current tab as private tab

### DISCUSSION THREAD

**Discussion by iambudi** [minbrowser/min#2044] [Ideas]
It would be handy if we just right click anywhere and click 'open current tab in/as new private tab'.

**Comment by PalmerAL** [comment]
Can you explain more what you would use this for?

It would be pretty easy to open a new private tab with the same URL; the problem is that we can't easily clear existing saved data from the non-private tab unless we cleared all data from that domain. Would this still be useful, even with that constraint?

**Reply by iambudi** [reply]
> Can you explain more what you would use this for?

1. Browsing anonymously from a page/domain since it's private :)
2. Workaround access for a site that has max page view limitation
3. For development purpose, compare page behaviour in non-private / private browsing

> It would be pretty easy to open a new private tab with the same URL; the problem is that we can't easily clear existing saved data from the non-private tab unless we cleared all data from that domain. Would this still be useful, even with that constraint?

Let's say i open google.com in non private tab with my google account already authenticated and when i open that in private mean my google account still identified not cleared?

**Reply by PalmerAL** [reply]
> Let's say i open google.com in non private tab with my google account already authenticated and when i open that in private mean my google account still identified not cleared?

If you open a non-private tab and sign into your google account, then right-click > "open in private tab", then in the private tab, you would not be signed in. But in the non-private tab, you still would be.

**Comment by iambudi** [comment]
Okay acceptable.
